src/Tribe/Linked_Posts.php

```diff
@@ -161,11 +161,11 @@ public function register_linked_post_type( $post_type, $args = array() ) {
 	 * @return string
 	 */
 	public function get_meta_key( $post_type ) {
-		if ( 'tribe_venue' === $post_type ) {
+		if ( Tribe__Events__Main::VENUE_POST_TYPE === $post_type ) {
```

Comment:
Lets use `Tribe__Events__Venue::POSTTYPE` constant, we are trying to deprecated this one.